kie-dmn/kie-dmn-feel/src/main/java/org/kie/dmn/feel/runtime/impl/RangeImpl.java

```diff
@@ -26,6 +33,39 @@
     private Comparable    lowEndPoint;
     private Comparable    highEndPoint;
 
+    public static RangeImpl of(EvaluationContext ctx, RangeBoundary lowBoundary, Object lowEndPoint, Object highEndPoint, RangeBoundary highBoundary) {
```

Comment:
I believe this method is not currently shared with the AST-interpreted,
so better to move into `CompiledFEELSemanticMappings` as sounds analogous to the FEEL list and other constructs there.

Reply:
ay ay captain

Reply:
thanks!